main_test.go

```diff
@@ -9,12 +9,64 @@ import (
 // TEST_DRIVERS: sqlite, mysql, postgres, sqlserver
 
 func TestGORM(t *testing.T) {
-	user := User{Name: "jinzhu"}
-
+	user := User{}
 	DB.Create(&user)
 
-	var result User
-	if err := DB.First(&result, user.ID).Error; err != nil {
-		t.Errorf("Failed, got error: %v", err)
+	pet1 := Pet{Name: "pet1", UserID: user.ID}
+	pet2 := Pet{Name: "pet2", UserID: user.ID}
```

Comment:
Here you're defining that user will have both pet1 and pet2, so there's no need to save it at line 38.

Reply:
👍 Thanks, I've deleted the User Save